### PR TITLE
Tp 464 fixup exporter tests

### PR DESCRIPTION
**Pull request by stuaxo**
Re-add exporter tests.
Catch up changes to docker-compose, Dockerfile.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,13 +43,8 @@ COPY --chown=tamato:tamato --from=jsdeps node_modules/govuk-frontend/govuk node_
...
 COPY --chown=tamato:tamato --from=jsdeps webpack-stats.json ./
 
-# empty .env file to prevent warning messages
-RUN touch .env
-
 # collect static files for deployment
-RUN python manage.py collectstatic --noinput
-
-ADD ./ /home/tamato/app/
+#RUN python manage.py collectstatic --noinput
```
Comment (NDevox) [inline Dockerfile:47]: Do we not want this now? We do have something of a frontend

**Review by NDevox — COMMENTED** [review]
All fine, just one q

**Review by NDevox — APPROVED** [review]
LGTM, squash commits though please!

File: Dockerfile (hunk 1 above)
Comment (stuaxo) [reply Dockerfile]: We do want this, I've just pushed the fix to re-enable it :)

Comment (stuaxo) [reply Dockerfile]: I knew something would go-awry patching this in the night.

**Comment by stuaxo** [comment]
> LGTM, squash commits though please!

Will do, hope this isn't because I forgot to do this before - luckily squash and merge seems to be the default :)

**Pull request merged**